contracts/VotingMachines/GenesisProtocol.sol

```diff
@@ -538,13 +539,14 @@ contract GenesisProtocol is IntVoteInterface,UniversalScheme {
      * @return int organization's score threshold.
      */
     function threshold(bytes32 _proposalId,address _avatar) public view returns(int) {
-        uint e = 2;
+        int88 e = 2;
         Parameters memory params = parameters[proposals[_proposalId].paramsHash];
-        uint power = orgBoostedProposalsCnt[_avatar]/params.thresholdConstB;
-        if (power > 100) {
```

Comment:
Just curious: Why has this changed from 100 to 86?

Reply:
The realMath lib works with int88 numbers

Reply:
Real math now works with int216 numbers so revert it back to 100 (which is huge enough number)